frontend_tests/node_tests/popovers.js

```diff
@@ -218,54 +214,3 @@ test_ui("sender_hover", ({override, mock_template}) => {
 
     // todo: load image
 });
-
-test_ui("actions_popover", ({override, mock_template}) => {
-    override($.fn, "popover", noop);
-
-    const $target = $.create("click target");
-
-    const handler = $("#main_div").get_on_handler("click", ".actions_hover");
-
-    const message = {
-        id: 999,
-        topic: "Actions (1)",
-        type: "stream",
-        stream_id: 123,
-        sent_by_me: true,
-    };
-
-    message_lists.current.get = (msg_id) => {
-        assert.equal(msg_id, message.id);
-        return message;
-    };
-
-    message_lists.current.view.message_containers.get = (msg_id) => {
-        assert.equal(msg_id, message.id);
-        return {
-            is_hidden: false,
-        };
-    };
-
-    override(page_params, "realm_allow_message_editing", true);
-    override(page_params, "realm_message_content_edit_limit_seconds", null);
-    assert.equal(message_edit.get_editability(message), message_edit.editability_types.FULL);
-
-    $target.closest = (sel) => {
-        assert.equal(sel, ".message_row");
-        return {
-            toggleClass: noop,
-        };
-    };
-
-    mock_template("actions_popover_template.hbs", false, () => "actions-template");
-    mock_template("actions_popover_content.hbs", false, (opts) => {
-        // TODO: Test all the properties of the popover
-        assert.equal(
-            opts.conversation_time_uri,
-            "http://zulip.zulipdev.com/#narrow/stream/123-unknown/topic/Actions.20.281.29/near/999",
-        );
-        return "actions-content";
-    });
-
-    handler.call($target, e);
-});
```

Comment:
I like the idea of opening a new issue for improving coverage of `popover_menus_data.js`; I don't think the old tests here were sufficiently good to be worth porting before merging this.